src/status_im2/contexts/communities/discover/style.cljs

```diff
@@ -56,8 +57,9 @@
   {:align-items     :center
    :justify-content :center})
 
-(def render-communities-container
-  {:margin-top 100})
+(defn render-communities-container
```

Comment:
Since the platform type is not reactive, we can still keep `render-communities-container` a static map.